### PR TITLE
chapter3: static_pages

### PR DESCRIPTION
**Pull request by gonepas**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Sampleapp</title>
+    <title><%= yield(:title) %> | <%= t "header" %></title>
     <%= csrf_meta_tags %>
-
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (NTHieu95) [inline app/views/layouts/application.html.erb]: Sua thanh double-quotes

File: Gemfile [hunk 2]
```diff
@@ -5,13 +5,14 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
```
Comment (NTHieu95) [inline Gemfile:6]: double-quotes

File: config/application.rb [hunk 3]
```diff
@@ -1,15 +1,8 @@
 require_relative 'boot'
```
Comment (NTHieu95) [inline config/application.rb]: double quotes

File: config/routes.rb [hunk 4]
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'static_pages/home'
```
Comment (NTHieu95) [inline config/routes.rb]: double quotes

File: config/application.rb [hunk 5]
```diff
@@ -1,15 +1,8 @@
...
-
 require 'rails/all'
```
Comment (NTHieu95) [inline config/application.rb]: double quotes

File: test/controllers/static_pages_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
```
Comment (NTHieu95) [inline test/controllers/static_pages_controller_test.rb]: duoble quotes

File: config/routes.rb [hunk 7]
```diff
@@ -1,3 +1,6 @@
...
+  get  'static_pages/about'
```
Comment (NTHieu95) [inline config/routes.rb]: xoa' 1 space

File: test/controllers/static_pages_controller_test.rb (hunk 6 above)
Comment (NTHieu95) [inline test/controllers/static_pages_controller_test.rb]: khong can up file nay` len dau

File: app/assets/javascripts/static_pages.coffee [hunk 8]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (hellovietnam93) [inline app/assets/javascripts/static_pages.coffee]: file này k dùng xóadid

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Sampleapp</title>
+    <title><%= yield(:title) %> | <%= t "header" %></title>
     <%= csrf_meta_tags %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track": "reload" %>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: 1 dấu cách ngăn thôi, quá 80 kí tự

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Sampleapp</title>
+    <title><%= yield(:title) %> | <%= t "header" %></title>
```
Comment (hellovietnam93) [inline app/views/layouts/application.html.erb]: xóa ()

**Pull request merged**